libvmaf/test/test_model.c

```diff
@@ -8,8 +7,16 @@ static char *test_model_load_and_destroy()
     int err;
 
     VmafModel *model;
-    err = vmaf_model_load_from_path(&model, "../../model/vmaf_v0.6.1.pkl");
+    VmafModelConfig config = {
```

Comment:
If you called `vmaf_model_config_destroy()` on this it would probably segfault since these are statically allocated.